src/App.tsx

```diff
@@ -1,61 +1,139 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+
+import { User } from './services/User';
+import { Todo } from './services/Todo';
+import { TodoWithUser } from './services/TodoWithUser';
+
+function getUserById(id: number): User | undefined {
+  return usersFromServer.find(user => user.id === id);
+}
+
+function newListWithUser(): TodoWithUser[] {
+  return todosFromServer.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
+  }));
+}
 
 export const App = () => {
+  const [listWithUser, setListWithUser] =
+    useState<TodoWithUser[]>(newListWithUser());
+  const [title, setTitle] = useState('');
+  const [choise, setChoice] = useState('0');
```

Comment:
```suggestion
  const [userId, setUserId] = useState('0');
```
type in choice word.

But again - better to rename it - we store userId here. Choice says nothing about this variable value